cdh4/impala/defaults

```diff
@@ -4,6 +4,7 @@ IMPALA_CATALOG_SERVICE_HOST={{ss_host}}
 IMPALA_STATE_STORE_PORT=24000
 IMPALA_BACKEND_PORT=22000
 IMPALA_LOG_DIR=/var/log/impala
+DEFAULT_POOL_MAX_REQUESTS={{ salt['pillar.get']('cdh4:impala:max_pool_requests', 20) }}
```

Comment:
Theoretically. Impala defaults it to 20, which is why i left that as the default in the file if no pillar data is provided.